src/components/atoms/Headline/Headline.tsx

```diff
@@ -0,0 +1,13 @@
+import { css } from "@emotion/css"
+
+export function Headline(props: { children: string }) {
```

Comment:
細かくて申し訳ないんだが、
コンポーネントのPropsの型は別で定義してほしい！
childrenの型はReact.ReactNodeを利用すると楽！
あと、functionではなくconstで統一したい！

```suggestion
type HeadlineProps = {
    children?: React.ReactNode
}

export const Headline: React.FC<HeadlineProps> = ({ children }) => {
```